integration-tests/gradle/src/test/java/io/quarkus/gradle/QuarkusPlatformAwareGradleTestBase.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.gradle;
+
+import java.io.IOException;
+import java.util.*;
+
+import io.quarkus.platform.descriptor.QuarkusPlatformDescriptor;
+import io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver;
+
+public class QuarkusPlatformAwareGradleTestBase extends QuarkusGradleWrapperTestBase{
```

Comment:
What is the point of this class? Was it copied from somewhere?

Reply:
Yes, it was copied from 
https://github.com/quarkusio/quarkus/blob/b697b25a95/integration-tests/devtools/src/test/java/io/quarkus/devtools/PlatformAwareTestBase.java
I commented about this above

Reply:
```getPluginGroupId() + ":" + getPluginArtifactId() + ":" + getPluginVersion()``` 
It was copied to have access to these, in line 31, but they aren't being used now

Reply:
I still don't understand what purpose it serves. How does this relate to the listExtensions test?

Reply:
The `addExtension ` tests don't use it, why does this test need it?

Reply:
@RotBolt sir can answer this, I'll remove it till then

Reply:
@geoand ListExtensions used PlatformAwareMaven. So I thought it would be good to have PlatformAwareGradle too it could serve base class for other future tests.

If not used in this test. May be could be considered in other PR or so

Reply:
So should I keep this file as it is and not extend it for this test ?

Reply:
We don;t need any of the maven specific stuff. The Gradle version of the ListExtensions test should be just like other Gradle versions of tests - unless there is a really good reason to diverge, and so far I don't see one